### DISCUSSION TITLE
Use Duration for timeouts

### DISCUSSION THREAD

**Discussion by nicos68** [hoodie/notify-rust#171] [Ideas]
I just found out about this lib. Thanks for working on it, it's  helpful for my current project.

One thing bugs me : shouldn't the `timeout()` function accept stdlib's ` Duration`  as an argument ? From what I can see it only takes a ` From`  implementation to allow it, and I think it would make things much more explicit and consistent, and avoid errors.

If you agree I can do the PR.

**Comment by hoodie** [comment]
Hey, thanks for the input, that is an excellent point!
Please go right ahead!

**Reply by nicos68** [reply]
[Done](https://github.com/hoodie/notify-rust/pull/172).
